### ISSUE TITLE
Custom study using tags with Image occlusion

### ISSUE THREAD

**Issue by Firewater21** [ankidroid/Anki-Android#16487]
### Checked for duplicates?

- [X] This issue is not a duplicate

### Does it also happen in the desktop version?

- [ ] This bug does not occur in the latest version of Anki

### What are the steps to reproduce this bug?

1. Make an image occlusion
2. Add a tag
3. Make a custom study
4. Limit to particular tags
5. Choose the tag of your image occlusion
6. Click all cards

### Expected behaviour

It shows all image occlusion cards. 

### Actual behaviour

It only shows the cards that it's due today. 

### Debug info

```text
AnkiDroid Version = 2.18.1 (5d13819ba811c6baea34dc747fce0568c755f31a)

Backend Version = 0.1.38-anki24.04.1 (24.04.1 ccd9ca1a8309b80bcb50ddc5d99c7ce63440bce9)

Android Version = 13 (SDK 33)

ProductFlavor = full

Manufacturer = Xiaomi

Model = M2102J20SG

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 13; M2102J20SG Build/TKQ1.221013.002; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/125.0.6422.53 Mobile Safari/537.36

ACRA UUID = afd6d8ba-7ead-4e46-8dc6-65488d3e4718

FSRS Enabled = true

Crash Reports Enabled = true
```

### (Optional) Anything else you want to share?

A video showing the behavior

https://github.com/ankidroid/Anki-Android/assets/149249012/ce54be4d-fc39-4d7a-9a81-c403dcb89218

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [X] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by Firewater21** [comment]
Update: It persist even in beta version 1

**Comment by Firewater21** [comment]
Update: It seems to be fixed in 2.19 beta 6. Thank you!

**Comment by david-allison** [comment]
Thanks so much for the update!

Saves us tons of time
